### ISSUE TITLE
OR_BIBED_07

### ISSUE THREAD

**Issue by GilTRipper** [busfor/react-native-google-pay#49]
I've implemented the library. But when I'm setting ENVIRONMENT_PRODUCTION - I have error OR_BIBED_07. I asked support they've said that there is an error with release key, but my release key the same as in console. 

Anyone faced with this?

**Comment by Amytrainer** [comment]
Yeah i am facing this , I believe this is to do with google pay api version which is 16. I have been banging my head around for some days but i am going to fork it now and test this as this is a possible problem. we need to have minumum to version 18 for wallet and payment services.

**Comment by KashifJavaid** [comment]
Yeah facing same and going to fork it and try solving this issue.

**Comment by asadMumtaz-YSTL** [comment]
> I've implemented the library. But when I'm setting ENVIRONMENT_PRODUCTION - I have error OR_BIBED_07. I asked support they've said that there is an error with release key, but my release key the same as in console.
> 
> Anyone faced with this?

Yeah facing same
@GilTRipper and @Amytrainer  you got any solution?

**Comment by itamarbareket** [comment]
Hi all,
fixed it here
https://github.com/mobimatter-com/react-native-google-pay

**Comment by kolindalanaresh** [comment]
@itamarbareket  unable to update to latest version still it is downloading 2.1.0 version and getting the same issue can you please help us.
